src/core/utils.js

```diff
@@ -277,6 +278,40 @@ export function joinAnd(array = [], mapper = item => item, lang = docLang) {
     }
   }
 }
+// STRING HELPERS
+// Takes an array and returns a string that separates each of its items with the proper commas and
+// "and". The second argument is a mapping function that can convert the items before they are
+// joined
+// Finally converts to hyperHTML after joining them
+export function joinAndHyper(
```

Comment:
Maybe `htmlJoinAnd`, because hyperHTML is just an implementation detail while the argument type and the return type is about `HTMLElement`s.